src/contracts/core/StrategyManager.sol

```diff
@@ -98,20 +98,17 @@ contract StrategyManager is
      * @param initialOwner Ownership of this contract is transferred to this address.
      * @param initialStrategyWhitelister The initial value of `strategyWhitelister` to set.
```

Comment:
some unused constants in this file i believe

Reply:
deleted 'em